.github/workflows/ci_cd.yml

```diff
@@ -38,17 +38,33 @@ jobs:
 
       - name: Install dependencies
         run: |
+          sudo apt update
+          sudo apt-get install -y build-essential zip pandoc texlive-latex-extra latexmk texlive-pstricks
           python -m pip install --upgrade pip tox
-      - name: Build documentation
+          python -m pip install -r requirements/requirements_doc.txt
+          python -m pip install --editable .
+
+      - name: Build HTML documentation
         run: tox -e doc
 
-      - name: Upload Documentation
+      - name: Upload HTML Documentation
         uses: actions/upload-artifact@v2.2.1
         with:
-          name: Documentation
+          name: HTML-Documentation
           path: .tox/doc_out/
           retention-days: 7
 
+      - name: Build PDF Documentation
```

Comment:
It would be interesting to include this step as a `tox` environment. We already tried but faced some problems. Let us discuss this in a separate issue.